src/services/beers.js

```diff
@@ -12,8 +13,42 @@ export async function fetchRandomBeer() {
   };
 };
 
-export async function fetchBeers(param) {
-  const url = `${baseUrl}/beers?${param.filter(String).join('&')}`
+export async function fetchBeersWithIds(param) {
+  const url = `${baseUrl}/beers?ids=${param}`
+  try {
+    const response = await fetch(url);
+    const beers = (await response.json())
+    return beers.map(
+      ({ id,
+        description,
+        name,
+        abv,
+        ibu,
+        ebc,
+        image_url: img,
+        tagline,
+        first_brewed: brewed,
+        food_pairing: food
+      }) => ({
+        id,
+        description,
+        ibu,
+        abv,
+        name,
+        ebc,
+        tagline,
+        img,
+        food,
+        brewed
+      })
+    )
```

Comment:
This logic repeats for `fetchBeersWithIds` and `fetchBeers`. Extract this to separate function.

Reply:
Done!